nmi_mysql/nmi_mysql.py

```diff
@@ -18,14 +18,13 @@
 
 class DB(object):
 
-    def __init__(self, conf, autoconnect=False):
+    def __init__(self, conf):
```

Comment:
I think if auto-connect is enabled, by default, we should just create one connection and put it in the pool.
If the user hits connect, then we just create a connection or return a new connection from the pool.